sql/src/main/java/io/crate/planner/node/dql/MergePhase.java

```diff
@@ -137,6 +138,49 @@ public static MergePhase sortedMerge(UUID jobId,
         return mergeNode;
     }
 
+    @Nullable
+    public static MergePhase mergePhase(ConsumerContext context,
```

Comment:
as far as I can see in the diff only the plannerContext of the ConsumerContext is accessed. So it would probably make sense to only depend on that.